tests/_sh/input-output/printf.c

```diff
@@ -1,12 +1,15 @@
+// Yash does not support * for precision like in printf "%.*s" 5 "abc"
+// printf_runtime works on yash because * are replaced with values before calling printf
+// expect_failure_for: yash
```

Comment:
Yash's `read` and `printf` functions are not POSIX compliant and so this test doesn't pass.